css/index.css

```diff
@@ -0,0 +1,280 @@
+@-webkit-keyframes confetti {
+  0% {
+    top: 100%;
+  }
+  100% {
+    top: -100%;
+  }
+}
+
+@keyframes confetti {
+  0% {
+    top: 100%;
+  }
+  100% {
+    top: -100%;
+  }
+}
+
+body {
+  margin: 0px;
+  padding: 1rem;
+  background:#0a182b;
+}
+
+.events-container {
+  margin: 0px;
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.events-parallax {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.events-parallax img {
+  position: absolute;
+  top: 0;
+  left: 50%;
+  width: 100%;
+  height: auto;
+  -webkit-transform: translateX(-50%) translateZ(0);
+  transform: translateX(-50%) translateZ(0);
+  -webkit-animation: confetti linear infinite;
+  animation: confetti linear infinite;
+}
+
+img:nth-child(1) {
```

Comment:
Let's add a class to these styles, so it doesn't effect other images.

Reply:
Done